public/js/shawarmaspin.js

```diff
@@ -216,6 +218,12 @@ angular.module('ShawarmaSpinApp', []).controller('ShawarmaController', ['$interv
 	Socket.io.on('score_minutes', function(data){
 		shawarma_ctrl.player.score_minutes = parseFloat(data);
 	});
+
+	/* On Startup */
+	if (window.location.hash && window.location.hash.length === 4){ // window.location.hash = #TST
```

Comment:
Why not just use Angular's route table like `/:team`. I think it's worth setting up in case we want to expand it to do a little more in the future.